network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/link_button_block.html

```diff
@@ -1,5 +1,8 @@
+{% extends "./base_streamfield_block.html" %}
 {% load wagtailcore_tags %}
 
-<div class="my-default">
-  <a class="btn {{ value.styling }}" href="{{ value.URL }}">{{ value.label }}</a>
+{% block block_content  %}
+<div class="my-2">
+  <a class="btn {{ value.styling }} link-button" href="{{ value.URL }}">{{ value.label }}</a>
 </div>
```

Comment:
It feels like we can probably get rid of that `<div>` and use `<a class="d-block my-2 btn ..."` instead